yang/vyatta-protocols-frr-bgp-v1.yang

```diff
@@ -321,6 +390,40 @@ module vyatta-protocols-frr-bgp-v1 {
 			}
 		}
 	}
+	grouping network-vpnv6-unicast {
+		list network {
+			description "BGP network";
+			configd:help "BGP network";
+			key "addr";
+			leaf addr {
+				description "BGP network";
+				type types:ipv6-prefix {
+					configd:normalize "normalize ipv4-prefix";
+				}
+				configd:help "BGP network";
+			}
+			leaf rd {
+				description "Specify Route Distinguisher";
+				configd:help "Specify Route Distinguisher";
+				configd:pattern-help "ASN:NN or IPAddress:NN";
+				type rt-rd;
+				default "100:1";
+			}
+			leaf label {
```

Comment:
setting a static vpn label should not be necessary. I don't think this should be exposed in the yang.